src/components/00-materials/package.json

```diff
@@ -20,10 +20,10 @@
     "type": "git",
     "url": "git+https://github.com/axa-ch/patterns-library.git"
   },
-  "scripts": {
-    "test": "echo \"Error: run tests from root\" && exit 1"
-  },
   "bugs": {
     "url": "https://github.com/axa-ch/patterns-library/issues"
+  },
+  "dependencies": {
```

Comment:
defently not a dependency but a devDependency

Reply:
Nice catch. done.

Reply:
done.